### DISCUSSION TITLE
sui publish 信息 没有package

### DISCUSSION THREAD

**Discussion by v1xingyue** [NonceGeek/Move-Camp#270] [Q&A]
sui官网介绍的 publish 信息如下: 

https://docs.sui.io/build/cli-client 

<img width="865" alt="image" src="https://user-images.githubusercontent.com/974169/216773488-8e5fa75f-d323-4648-a44c-a2b6d84c528a.png">

我本地发布的信息如下: 

<img width="1197" alt="image" src="https://user-images.githubusercontent.com/974169/216773526-b70564a9-3e11-4960-9bbe-60bce8e3822b.png">

并没有package 信息。 

当然，我可以拿着 Transaction 去 浏览器里自己找，只是好奇 是把这些提示删掉了么？ 你们的有这个 The newly published package object 么？

**Comment by RandyPen** [comment]
<img width="490" alt="image" src="https://user-images.githubusercontent.com/57551060/216773689-3b30adaf-e667-48e5-a783-b49baf35bc33.png">
这个就是你的package id.      

immutable 不可变，属于你的账户发布

**Comment by qiwihui** [comment]
补充： 可以用 `sui client object {object_id}` 来查看对应 object 的信息。
